oio/xcute/orchestrator.py

```diff
@@ -58,28 +59,21 @@ def __init__(self, conf, verbose):
                          self.beanstalkd_reply_tube)
 
         self.running = True
-        self.threads = {}
+        self.beanstalkd_workers = None
+        self.beanstalkd_workers_next_index = 0
```

Comment:
Can't we make `beanstalkd_workers` an iterator instead of this random global state?